app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md

```diff
@@ -0,0 +1,321 @@
+
+[Learn more about applying for a visa to train to teach in England](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#visa).
+
+#### How the IRP will be paid
+
+If you have a place on a teacher training course where you pay a fee, you do not need to apply for the IRP. Your training provider will email you if your choice of course makes you eligible for the IRP. Your visa status and date of arrival will be checked by DfE. If you meet the eligibility requirements, you’ll get the IRP around the end of your first term.
+
+If you have a place on a teacher training course where you are paid a salary to work while you train, email us at IRP.ExpressInterest@education.gov.uk to express your interest in the IRP. We’ll be in touch when IRP applications open for salaried trainees in autumn 2023. If your application is successful, you’ll get the IRP around the end of your first term.
+
+You will not need to pay an agent or any other third party to complete your application. If you have had a message asking you to apply for or contact someone about the IRP, [this could be fraudulent](https://www.gov.uk/report-suspicious-emails-websites-phishing).
+
+### Bursaries and scholarships for physics and languages
+
+Bursaries and scholarships are cash sums from the UK government that you do not have to pay back. Bear in mind that:
+
+* you cannot be awarded both a bursary and a scholarship
+* bursaries and scholarships are only available for courses where you pay a fee to study teacher training
+* bursaries and scholarships in physics and languages are open to all non-UK trainees (even if you are not eligible for other types of financial support from the UK government)  
+
+#### Bursaries for physics and languages
+
+You do not need to apply for a bursary – it will automatically be awarded to you through your training provider if you are eligible.
+
+You may get a:
+
+* physics bursary of £27,000
+* languages bursary of £25,000
+
+To be eligible, you’ll need a place on a course training to teach languages or physics with an accredited teacher training provider in England, in the academic year 2023 to 2024.
+
+You’ll also need at least one of the following:
+
+* a bachelor’s degree class 2:2 (honours) or higher
+* a master’s degree
+* a PhD
+
+Each bursary scheme will have their own individual terms and conditions – contact your chosen teacher training provider for more information.
+
+The languages bursary covers all languages offered by accredited teacher training providers. If a course combines 2 subjects (for example, physics plus mathematics, or modern foreign languages plus English), at least 50% of the course must be made up of the bursary subject.
+
+
+#### Scholarships for physics, French, German and Spanish
+
+To get a scholarship, you’ll need to be a high-quality student and pass a tough selection process, run by the professional body for your subject.
```

Comment:
```suggestion
To get a scholarship, you’ll need to be a high-quality student and pass a tough selection process.
```
Will users understand what is meant by 'professional body'? Do they need this bit of information?